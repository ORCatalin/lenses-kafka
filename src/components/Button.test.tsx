import React from 'react';
import { render, screen, fireEvent } from '@testing-library/react';

import Button from "./Button";

describe('Button component', () => {
  test('is rendering', () => {
    const { asFragment } = render(
      <Button>Test</Button>
    );
    expect(asFragment()).toMatchSnapshot();
  })
});
