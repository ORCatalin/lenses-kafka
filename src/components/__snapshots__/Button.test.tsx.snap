// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button component is rendering 1`] = `
<DocumentFragment>
  <button
    type="button"
  >
    Test
  </button>
</DocumentFragment>
`;
